src/PoolManager.sol

```diff
@@ -39,10 +39,12 @@ contract PoolManager is IPoolManager, Fees, NoDelegateCall, ERC6909Claims {
     /// @inheritdoc IPoolManager
     int24 public constant MIN_TICK_SPACING = TickMath.MIN_TICK_SPACING;
 
+    uint256 public constant TOTAL_DEBT = uint256(0);
```

Comment:
nit: would appreciate a comment here saying that this is a sentinel value for target's entire delta balance

Reply:
would max uint be better here?

Reply:
0 is much cheaper calldata so its what we use [in UR](https://github.com/Uniswap/universal-router/blob/41183d6eb154f0ab0e74a0e911a5ef9ea51fc4bd/contracts/libraries/Constants.sol#L11), and [SwapRouter02](https://github.com/Uniswap/swap-router-contracts/blob/550c0f20373a487996fcc957075377b67af9df07/contracts/libraries/Constants.sol#L8) to signal "the entire value".
However happy to discuss if you have a strong reason for changing!